src/pages/en/install/manual.md

```diff
@@ -146,7 +146,26 @@ If you want to include [UI framework components](/en/core-concepts/framework-com
 
 📚 Read Astro's [API configuration reference](/en/reference/configuration-reference/) for more information.
 
-## 6. Next Steps
+## 6. Create `tsconfig.json`
+
+Typescript is configured using `tsconfig.json`. Even if you don’t write TypeScript code, this file is important so that tools like Astro and VS Code know how to understand your project. Some features (like npm package imports) aren’t fully supported in the editor without a `tsconfig.json` file. 
+
+If you do intend to write TypeScript code, using Astro's `strict` or `strictest` template is recommended. You can view and compare the three template configurations at [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/).
```

Comment:
_⚠️ here be minutiae! 🔬_

This is so minor, but part of me wants to change `[astro/tsconfigs/]` to `[astro/tsconfigs]` (remove trailing slash from link text).

Looking through this repo (VS Code), comparing existing usage:

Search: ``\[[^ `@<>\]]+/[^ `@<>\]]+[^/]\]\(``
files to include: `*.md`
files to exclude: `old-translations`
79 results in 20 files.

Search: `\[[^ `@<>\]]+/[^ `@<>\]]+/\]\(`
files to include: `*.md`
files to exclude: `old-translations`
15 results in 15 files:
- 8 results: `[http://localhost:3000/](`
- 4 results: `[gohugo.io/documentation/](`
- 3 results: `[astro/tsconfigs/](`

Off topic for this PR, but how about a simple rule for this, something like:
>Use a trailing slash in link text when:
>1. It adds semantic meaning: "oh, this link leads to a directory".
>2. Important for technical clarity.

Based on that, case 1 would apply to `[astro/tsconfigs/](`, so the trailing slash would remain. Neither 1 or 2 would apply to `[gohugo.io/documentation/](`, so that trailing slash would be dropped. I would personally say to 2 doesn't strongly apply to `[http://localhost:3000/](`, but given that the link text is the same as the link URL, it should stay.